src/app/technical-record/vehicle-summary/vehicle-summary-psv/vehicle-summary-psv.component.html

```diff
@@ -66,4 +66,4 @@
   <dl class="govuk-summary-list"></dl>
 </dl>
 
-<ng-template #hyphen><span>-</span></ng-template>
+<ng-template #hyphen><span class="govuk-body">-</span></ng-template>
```

Comment:
should we have this `govuk-body`

Reply:
This was an attempt to not use important :)

Reply:
removed